### ISSUE TITLE
publisher and Identifier blocks are inconsistent

### ISSUE THREAD

**Issue by Bjwebb** [open-contracting/standard#245]
`publisher` in both the release and record package schemas has:
`scheme` `uid` `name`  and `uri`

Identifier has:
`scheme` `id` `legalName` and `uri`

`scheme` and `uri` are the same.
Does `uid`=`id` and `name`=`legalName` ?

If so, should the fields in the publisher block be renamed? This would be a backwards incompatible change.

**Comment by timgdavies** [comment]
I've flagged this for inclusion in the 1.1 revisions. 

I would support aligning these. 

We should add a note to the documentation in the meantime.

**Comment by timgdavies** [comment]
The `scheme` field for `publisher` in the release and record package also includes a `"format":"uri"` property, which incorrectly causes format validation to reject string scheme values.

This should be fixed in 1.0.1 bug fix release.

**Comment by Bjwebb** [comment]
I don't think this has been fixed. The change in 1.0.1 doesn't address the underlying issue.

**Comment by timgdavies** [comment]
Ah - sorry. Yep. Is there a non backwards-incompatible update we can make here in 1.1?

**Comment by Bjwebb** [comment]
I'm not sure there is for 1.1, as renaming is backwards incompatible. There's the option of adding the new "correct" fields and deprecating the older ones, but that's probably more confusing than the current situation.

**Comment by andrewlorien** [comment]
in NSW eTendering and AusTender, suppliers have "trading name" as well as "legal name".  Currently I'm mapping "tradingName" to "Name", so this change would actually affect my data.
If publisher and identifier both had "name" AND "legalName", that field wouldn't be a breaking change.

**Comment by timgdavies** [comment]
This was missed from 1.1. Needs action in future revision.

**Comment by jpmckinney** [comment]
I'm not sure the pain of inconsistency is greater than the pain of upgrading here.

**Comment by jpmckinney** [comment]
Closing - see reason in #434.
